sherpa/__init__.py

```diff
@@ -116,11 +116,25 @@ def test(level=1, verbosity=1, datadir=None):
 
 
 def clitest():
-    try:
-        import pytest, pytest_cov
-        errno = pytest.main(['--pyargs', 'sherpa', '-rs'])
-        sys.exit(errno)
-    except ImportError:
-        print """Cannot import pytest and pytest-cov.
-            Please run 'pip install -r test_requirements.txt' first"""
-        sys.exit(1)
+    def install(package_name):
+        try:
```

Comment:
Would it be worth having separate try/except blocks for `import pip` and `pip.main` so that a user can identify what failed? Or is there going to be enough screen output that you can work it out?

This is not a strong suggestion.